keyboards/id87/readme.md

```diff
@@ -0,0 +1,15 @@
+# IDOBAO ID87
+
+A TKL hotswap board from IDOBAO.
+
+* Hardware Supported: IDOBAO ID87
+
+Hardware URL: [Idobao](https://www.idobao.net/products/idobao-id87-80-hot-swappable-mechanical-keyboard-kit?variant=33990062604419)
+
+This board can enter DFU mode via FN(The menu button)+ESC
+
+Make example for this keyboard (after setting up your build environment):
+
+    make id87:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Please update this to match [`quantum/template/avr/readme.md`](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md) (maybe you used an older version of template).

You could probably drop the `?variant=33990062604419` part from the URL.